scss/bitstyles/layout/_width.scss

```diff
@@ -11,10 +11,10 @@
 // `.l-width--1-of-12` … `.l-width--12-of-12`.
 //
 // If you define `$bitstyles-widths-breakpoints` to contain the names of one or more
-// breakpoints (defaults to just `medium-and-up`), widths will also be created
+// breakpoints (defaults to just `medium`), widths will also be created
 // for those breakpoints:
 //
-// `.l-width--4-of-12-medium-and-up`
+// `.l-width--4-of-12-medium`
```

Comment:
I’d say just leave a line comment when making your review — the PR author can fix it/suggest something else and update their PR.

But in this case yup that’s fine :)